corehq/apps/products/models.py

```diff
@@ -297,7 +297,7 @@ class SQLProduct(models.Model):
     name = models.CharField(max_length=100, null=True)
     is_archived = models.BooleanField(default=False)
     code = models.CharField(max_length=100, default='', null=True)
-    description = models.CharField(max_length=100, null=True, default='')
+    description = models.TextField(null=True, default='')
```

Comment:
The rest of these fields are a bit strange too - the 100 character limit is entirely arbitrary, as far as I can tell (I think 255 characters will be allocated anyways behind the scene), and [the docs](https://docs.djangoproject.com/en/1.5/ref/models/fields/#null) say you shouldn't set null to true on text or char fields, because then they'll be stored as `NULL` instead of the empty string.  We might take this opportunity to clean up all of the field definitions here, but that feels like a bit of scope creep given what triggered thsi change, so I'll table it 'till I actually touch Products, unless @twymer feels strongly